9.2-2.3/alpine/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM postgres:9.2-alpine
+MAINTAINER Régis Belson <me@regisbelson.fr>
+
+ENV POSTGIS_VERSION 2.3.1
+ENV POSTGIS_SHA256 0f1e0fe08f55b6db791d8600bb343e4f77590725acbbec99094a30b8b7439cd8
+
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main >> /etc/apk/repositories
```

Comment:
Since the `postgres:alpine` base images are not based on `alpine:edge`, do you think you could explain this addition along with the call to `apk upgrade` below?